File: client/job.php
<?php

class Job {
    public $name;
    public $url;
    function __construct($name, $url)
    {
        $this->name = $name;
        $this->url = $url;
    }

    public function getDescription() {
        return 'jkljlkjlk ' . $this->name . 'jkjkljkljkljlk' . $this->url;
    }
}


$job = new Job('jan', 'jkljkljkl');

echo $job->getDescription();

File: client/job.go
package client

import "time"

type Job struct {
	Name  string
	URL   string
	Birth time.Time
}

func (j *Job) GetDescription() string {
	return "job named :" + j.Name + " is available at :" + j.URL
}

func NewJob(name string, url string) *Job {
	return &Job{
		Name:  name,
		URL:   url,
		Birth: time.Now(),
	}
}
